kaldb/src/main/java/com/slack/kaldb/metadata/snapshot/SnapshotMetadata.java

```diff
@@ -29,7 +52,7 @@ public SnapshotMetadata(
     checkState(startTimeUtc > 0, "start time should be greater than zero.");
     checkState(endTimeUtc > 0, "end time should be greater than zero.");
     checkState(endTimeUtc > startTimeUtc, "start time should be greater than  end time.");
-    checkState(maxOffset > 0, "max offset should be greater than zero.");
+    checkState(maxOffset >= 0, "max offset should be greater than zero.");
```

Comment:
We need to update the error message as well

Reply:
Done.